Signal/src/ViewControllers/DebugUI/DebugUIMessages.m

```diff
@@ -25,6 +25,13 @@
 
 NS_ASSUME_NONNULL_BEGIN
```

Comment:
Nothing to do with this PR, but this Debug UI page has been so useful that it has grown unwieldy.  I intend to eventually clean it up, especially by DRYing repetitive logic.